CHANGELOG.md

```diff
@@ -2,6 +2,12 @@ Note: This is in reverse chronological order, so newer entries are added to the
 
 Swift 5.6
 -----------
+
+* [#3890]
```

Comment:
sorry one more observation: the PRs are listed by order so this should come last in the 5.6 section

Reply:
So releases are in reverse chronological order but within a release they are in ascending?

Reply:
amended

Reply:
looks like the PRs themselves are not actually in order :/ but the SEs are listed before PRs